validations/validations_test.go

```diff
@@ -1,13 +1,17 @@
 package validations_test
 
 import (
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/Betterment/testtrack-cli/serializers"
 	"github.com/Betterment/testtrack-cli/validations"
 	"github.com/stretchr/testify/require"
 )
 
+const defaultOwnershipFilename = "testtrack/owners.yml"
```

Comment:
I'm new to go -- got a bit stuck trying to figure out how to make this a shared variable between `validations.go` and `validations_test.go`